cli/tests/test_commit_template.rs

```diff
@@ -146,6 +146,35 @@ fn test_log_author_timestamp_local() {
     "###);
 }
 
+#[test]
+fn test_mine_is_true_when_author_is_user() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    test_env.jj_cmd_ok(
+        &repo_path,
+        &[
+            "--config-toml=user.email='johndoe@example.com'",
+            "--config-toml=user.name='John Doe'",
+            "new",
+        ],
+    );
+
+    let stdout = test_env.jj_cmd_success(
+        &repo_path,
+        &[
+            "log",
+            "-T",
+            r#"if(mine, "mine", coalesce(author.email(), email_placeholder))"#,
```

Comment:
nit: `if` statements can be one-armed, so the coalesce can be floated outward:

```
r#"coalesce(if(mine, "mine"), author.email(), email_placeholder)"#,
```

I find this cleaner. YMMV.